airflow/exceptions.py

```diff
@@ -424,3 +424,27 @@ def __str__(self):
             "Error deserializing result. Note that result deserialization "
             "is not supported across major Python versions. Cause: " + str(self.__cause__)
         )
+
+
+class DockerContainerFailedException(AirflowException):
```

Comment:
Since these exceptions are only used by one provider (one operator really), they should be defined in that operator file instead.

Reply:
Good point - just moved them there.